src/main/scala/au/com/agiledigital/toolform/app/ToolFormApp.scala

```diff
@@ -40,7 +41,7 @@ object CliParserConfiguration {
     * @return Opts style command line parser options for use in a decline
     *         parser/app runner.
     */
-  def commandLineOptions: Opts[Either[ToolFormError, String]] =
+  def commandLineOptions: Opts[Either[NonEmptyList[ToolFormError], String]] =
```

Comment:
`Either[NEL[ToolFormError], String]]` it becoming a good candidate for a type def. No change required for this PR though.

Reply:
👍